worker/src/tokio_handle_accessor.rs

```diff
@@ -0,0 +1,61 @@
+/*
+    Copyright 2019 Supercomputing Systems AG
+    Copyright (C) 2017-2019 Baidu, Inc. All Rights Reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+use lazy_static::lazy_static;
+use std::sync::Mutex;
+use tokio::runtime::Handle;
+
+lazy_static! {
+    static ref TOKIO_HANDLE: Mutex<Option<tokio::runtime::Handle>> = Default::default();
+}
+
+/// Wrapper for accessing a tokio handle
+pub trait TokioHandleAccessor {
```

Comment:
`GetTokioHandle` is a naming pattern for a method, because it contains a verb. A trait or class should be a noun, that's why I chose `accessor`. We could call it `TokioHandleGetter` - but I'm not sure that would be an improvement.